src/Stats.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <QLineEdit>
+#include <QMap>
+#define UNSOLVED_SCORE 2000
+
+namespace mms {
+
+enum class StatsEnum{TOTAL_DISTANCE,
+               TOTAL_TURNS,
+               BEST_RUN_DISTANCE,
+               BEST_RUN_TURNS,
+               CURRENT_RUN_DISTANCE,
+               CURRENT_RUN_TURNS,
+               TOTAL_EFFECTIVE_DISTANCE,
+               BEST_RUN_EFFECTIVE_DISTANCE,
+               CURRENT_RUN_EFFECTIVE_DISTANCE,
+               SCORE // has a text box but is not saved in an array
+              };
+
+class Stats
+{
+public:
+    Stats();
+    void resetAll(); // Reset all score stats
+    void addDistance(int distance = 1); // Increase the distance and effective distance
```

Comment:
Nitpick: don't specify a default value for `distance` - there's no need, since the caller always provides one.